.github/workflows/ci.yaml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on: [workflow_dispatch, pull_request]
+
+env:
+  SECRET_KEY: thisisasecret
+  DATABASE_URL: postgresql://postgres:password@localhost:5432/postgres
+
+jobs:
+  ci:
+    strategy:
+      matrix:
+        python-version: ["3.6"]
+        platform: [ubuntu-20.04]
+
+    runs-on: ${{ matrix.platform }}
+
+    services:
+      postgres:
+        image: postgres:11-alpine
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: password
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+
+    - name: Set up Python dependencies
+      run: |
+        pip install -U pip
+        # pip install wheel
+        # pip wheel -q -r requirements/travis.txt
+        pip install -r requirements/travis.txt
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 12
```

Comment:
Can the node version be set to v8? Though it's pretty old, [we still use it in production](https://github.com/pycontw/pycon.tw/blob/master/Dockerfile#L10).